app/models/concerns/archivable.rb

```diff
@@ -0,0 +1,12 @@
+module Archivable
+  extend ActiveSupport::Concern
+
+  included do
+    scope :archived, -> { where('archived_at is not null') }
+    scope :published, -> { where(archived_at: nil) }
```

Comment:
Arrr also.. I just realised that we won't be considering archived when pulling a list of comments, because we want to include archived comments in the feed. So insted we should have a body method that returns blank if the comment is archived. We'll need this so that the CommentSerializer does not return the comment body for angular users.